ports/servoshell/desktop/headed_window.rs

```diff
@@ -72,11 +72,20 @@ impl Window {
         // #9996.
         let visible = opts.output_file.is_none() && !no_native_titlebar;
 
+        // Do not let the window size start larger than the screen size override.
```

Comment:
Is this constraint strictly required? Is it not valid to create a window wider than the screen, for example, to span two monitors in extended display configuration, or is this just for convenience (so we don't have to pass --window-size for simulation of mobile screens)?

Reply:
It's not strictly required, but it makes setting the screen size a lot friendlier as you also don't have to modify the window size. I've changed this code so it only affects the default window size. This allows you to specify a window size larger than the screen, if you do it manually.